src/gradata/integrations/crewai_adapter.py

```
@@ -101,4 +101,4 @@ def get_rules(self, task: str = "", context: dict | None = None) -> str:
         try:
             return self.brain.apply_brain_rules(task, context)
         except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)